test/.eslintrc.json

```diff
@@ -1,4 +1,6 @@
 {
+	"extends": "../src/.eslintrc.json",
+	"root": true,
```

Comment:
Hm, how about we extend only `eslint-jquery-config` with `root: true` in every config? It will cause some repetition, but it could make it very clear. What do you think?